index.html

```diff
@@ -384,11 +384,94 @@ <h2>Key Discovery</h2>
       <p class="issue">
         The working group is still discussing how to close many related issues.
       </p>
-      <p class="issue" data-number="117"></p>
-      <p class="issue" data-number="106"></p>
-      <p class="issue" data-number="31"></p>
-      <p class="issue" data-number="30"></p>
-      <p class="issue" data-number="15"></p>
+
+      <!-- DID URLS via "issuer" and "holder" -->
+      <p>
+        When the <a data-cite="VC-DATA-MODEL#dfn-issuers">issuer</a> is identified as a <a data-cite="DID-CORE#did-subject">DID Subject</a>, 
+        the <a href="#kid">kid</a> MUST be an absolute <a data-cite="DID-CORE#relative-did-urls">DID URL</a>.
+      </p>
+<pre class="example" title="An issuer identified by a DID">
+{
+  "issuer": "did:example:123"
+  // ...
+}
+</pre>
+<pre class="example" title="An absolute DID URL as a kid">
+{
+  "alg": "ES384",
+  "kid": "did:example:123#key-456
+}
+</pre>
+      <p>
+        When the <a data-cite="VC-DATA-MODEL#dfn-holders">holder</a> is identified as a <a data-cite="DID-CORE#did-subject">DID Subject</a>, 
+        the <a href="#kid">kid</a> MUST be an absolute <a data-cite="DID-CORE#relative-did-urls">DID URL</a>.
+      </p>
+<pre class="example" title="A holder identified by a DID">
+{
+  "holder": "did:example:abc"
+  // ...
+}
+</pre>
+<pre class="example" title="An kid as an absolute DID URL">
+{
+  "alg": "ES384",
+  "kid": "did:example:abc#key-456
+}
+</pre>
+
+      <!-- REGULAR URLS via "issuer" and "holder" -->
+      <p>
+        When the <a data-cite="VC-DATA-MODEL#dfn-issuers">issuer</a> is identified as a [[URL]], 
+        the <a href="#kid">kid</a> MUST be an absolute [[URL]] to a verification method listed in a controller document.
```

Comment:
Can "controller document" be a reference to a defined term?

Reply:
sadly not yet, this is due to the dialog regarding data integrity defining controller document.

Reply:
See: https://github.com/w3c/vc-jose-cose/issues/160